src/client/dfuse/SConscript

```diff
@@ -116,6 +116,19 @@ def check_struct_member(env, text, struct, member):
     env.Result(rc)
     return rc
 
+def configure_fuse(env):
+    """Configure for specific version of fuse"""
+    if GetOption('help'):
+        return
+
+    check = Configure(cenv, custom_tests={'CheckStructMember': check_struct_member})
+
+    if check.CheckStructMember('#include <fuse3/fuse.h>',
+                               'struct fuse_file_info',
+                               'cache_readdir'):
+        cenv.AppendUnique(CPPDEFINES=['-DHAVE_CACHE_READDIR=1'])
```

Comment:
(pylint-undefined-variable) Undefined variable 'cenv'

Reply:
I'm confused by this one, is it a false positive?